docs/advanced.rst

```diff
@@ -739,278 +740,401 @@ Thus, for a higher performance, it can be appropriate to configure the reliabili
-  | **XML**                                             |
-  +-----------------------------------------------------+
-  | .. literalinclude:: ../code/XMLTester.xml           |
-  |    :language: xml                                   |
-  |    :start-after: <!-->CONF-QOS-DISCOVERY-SERVERLIST |
-  |    :end-before: <!--><-->                           |
-  +-----------------------------------------------------+
-
-+ **Duration_t leaseDuration**. Linked with the participant *liveliness mechanism* it specifies how much time other
-  remote participants should consider this one alive.
-
-+ **Duration_t leaseDuration_announcementperiod**. Linked either with participant *liveliness and announcement
-  mechanisms*. It specifies how often a participant should send its discovery data in order to notify new participants
-  and refresh old ones *liveliness*:
-
- - For liveliness sake it should be smaller than the *leaseDuration* in order to avoid other participants to kill this
-   one.
-
- - There is a trade-off involved with the announcement. Too frequent announcement will bloat the network with
-   metatraffic but too scarce ones will delay the discovery of late joiners.
-
-  +--------------------------------------------------------+
-  | **C++**                                                |
-  +--------------------------------------------------------+
-  | .. literalinclude:: ../code/CodeTester.cpp             |
-  |    :language: c++                                      |
-  |    :start-after: //CONF-QOS-DISCOVERY-LEASEDURATION    |
-  |    :end-before: //!--                                  |
-  +--------------------------------------------------------+
-  | **XML**                                                |
-  +--------------------------------------------------------+
-  | .. literalinclude:: ../code/XMLTester.xml              |
-  |    :language: xml                                      |
-  |    :start-after: <!-->CONF-QOS-DISCOVERY-LEASEDURATION |
-  |    :end-before: <!--><-->                              |
-  +--------------------------------------------------------+
-
-+ **Duration_t discoveryServer_client_syncperiod**. Linked either with client's *announcement mechanism*. It specifies
-  how often a client should send its discovery data to a server that is not yet aware of its presence. Given the fact
-  that this announcement will be shut down the very moment the server acknowledges the client, its frequency will not
-  be a concern for network traffic outside network initialization stage.
-
- Note that servers act like clients whenever they reference other servers, thus, *discoveryServer_client_syncperiod*
- applies to them also in this scenario.
+.. _discovery_lease_announ:
+
+Announcement Period
+-------------------
+
+It specifies the periodicity of the participant's PDP announcements.
+For liveliness shake it is recommend to be shorter than the lease duration, so that the participant's liveliness is
```

Comment:
```suggestion
For liveliness' sake it is recommended that the announcement period is shorter than the lease duration, so that the participant's liveliness is
```

Reply:
ok